Process Process-3:
Traceback (most recent call last):
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "batchComment.py", line 147, in commentProcess
    flag = self.comment(mids, st)
  File "batchComment.py", line 139, in comment
    self.IAmHuman()
  File "batchComment.py", line 66, in IAmHuman
    res = session.get(bhv, headers=headers)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 487, in get
    return self.request('GET', url, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 585, in send
    r = adapter.send(request, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/adapters.py", line 479, in send
    raise ReadTimeout(e, request=request)
ReadTimeout: HTTPConnectionPool(host='10.131.16.74', port=8000): Read timed out. (read timeout=None)
Process Process-2:
Traceback (most recent call last):
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "batchComment.py", line 147, in commentProcess
    flag = self.comment(mids, st)
  File "batchComment.py", line 139, in comment
    self.IAmHuman()
  File "batchComment.py", line 66, in IAmHuman
    res = session.get(bhv, headers=headers)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 487, in get
    return self.request('GET', url, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 585, in send
    r = adapter.send(request, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/adapters.py", line 479, in send
    raise ReadTimeout(e, request=request)
ReadTimeout: HTTPConnectionPool(host='10.131.16.74', port=8000): Read timed out. (read timeout=None)
Process Process-4:
Traceback (most recent call last):
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "batchComment.py", line 147, in commentProcess
    flag = self.comment(mids, st)
  File "batchComment.py", line 139, in comment
    self.IAmHuman()
  File "batchComment.py", line 66, in IAmHuman
    res = session.get(bhv, headers=headers)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 487, in get
    return self.request('GET', url, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 585, in send
    r = adapter.send(request, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/adapters.py", line 479, in send
    raise ReadTimeout(e, request=request)
ReadTimeout: HTTPConnectionPool(host='10.131.16.74', port=8000): Read timed out. (read timeout=None)
Process Process-1:
Traceback (most recent call last):
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/Users/jiawei/anaconda/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "batchComment.py", line 147, in commentProcess
    flag = self.comment(mids, st)
  File "batchComment.py", line 135, in comment
    edit_status = self.postComment(uid, mid, comment_content, st1)
  File "batchComment.py", line 91, in postComment
    res = session.post(url, data=form_data, headers=headers)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 518, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 475, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 585, in send
    r = adapter.send(request, **kwargs)
  File "/Users/jiawei/anaconda/lib/python2.7/site-packages/requests/adapters.py", line 479, in send
    raise ReadTimeout(e, request=request)
ReadTimeout: HTTPConnectionPool(host='10.131.16.74', port=8000): Read timed out. (read timeout=None)
